app/components/util/HowDoYouFeel.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react'
```

Comment:
This file should be moved up a directory, and replace the one that's already there from before.  Also, must use lower case characters and dasher (kebob case it's called) for file names.